Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+MAINTAINER Devin Ekins <devops@keen.io>
+
+# Setup the dependencies
+RUN apt-get update -y
+RUN apt-get install -y apt-utils
```

Comment:
Still would follow the recommendations they make and work to minimize the number of RUN commands.